### ISSUE TITLE
I think this one is rude for hongkong people and kind of bandalism. I don't want to understand your motivation, but if you want to make this change, first remove Hong Kong and Macau from the [country code list](https://en.wikipedia.org/wiki/List_of_country_calling_codes).

### ISSUE THREAD

**Issue by fishballzzz** [ffaker/ffaker#479]
I think this one is rude for hongkong people and kind of bandalism. I don't want to understand your motivation, but if you want to make this change, first remove Hong Kong and Macau from the [country code list](https://en.wikipedia.org/wiki/List_of_country_calling_codes).

_Originally posted by @marocchino in https://github.com/ffaker/ffaker/issues/477#issuecomment-785498775_

Hey, I am just stating a fact that it's not a country. Having a special area code doesn't make them a country.
You can find a bunch of official documentations that it's not a country. And it will never be rude to them even if they want to deny that. Or you can just trust solely on Wikipedia.
As a user, thank you again for putting effort in maintaining this library. You might have valid reasons to list them as countries, but you can compare how faker does it.
I won't go further to debate on this. You can think twice on your decisions.

**Comment by rstacruz** [comment]
The list in the gem tracks the country codes listed in [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) <https://www.iso.org/obp/ui/#search>.

**Comment by marocchino** [comment]
Thank you for your explanation. 

> Having a special area code doesn't make them a country.

If so, your modification should have been to change the country code to the area code not delete them. Although the results were similar but less rude.

> You can think twice on your decisions.
Even if I think twice, this doesn't seem to be the right place for this discussion. Go to ISO and protest about it.

**Comment by marocchino** [comment]
For clarity, I am neither affirming nor denying the proposition that Hong Kong is a country, but I think it is a vandalism you try to erase part of list for the political reason.

**Comment by fishballzzz** [comment]
Just to clarify, I wasn't trying to sabotage this repo for political reasons. I merely think that when I was using the library, I saw country, state, city for location settings and HK, MO were listed as countries not states, which caused confusions hence reporting the issue. 
The PR may not be a good solution as I don't know enough to make the change, but I was just giving it a try as asked. I had mentioned nothing political there, but somehow you think my PR is rude for HK people. I suggested at the beginning, they should be listed as **states** instead of _countries_. 
On the ISO page, the definition for a country includes both an independent sovereign state or a part of a larger independent sovereign state, and I think that they are more closed to the definition of a state, especially when most people select locations following the pattern country>state/province>city.
